lib/Scheduler/bb_spill.cpp

```diff
@@ -809,20 +820,21 @@ FUNC_RESULT BBWithSpill::Enumerate_(Milliseconds startTime,
       (rgnTimeout == INVALID_VALUE) ? INVALID_VALUE : startTime + rgnTimeout;
   lngthDeadline =
       (rgnTimeout == INVALID_VALUE) ? INVALID_VALUE : startTime + lngthTimeout;
-  assert(lngthDeadline <= rgnDeadline);
+  // assert(lngthDeadline <= rgnDeadline);
```

Comment:
Why was this made commented out? Why not keep it?

Reply:
This represents a bug in the master branch (as I understand it) to be addressed in a separate issue. 

The effect of this assert is to halt runtime if the config file specifies a `REGION_TIMEOUT` that is less than `LENGTH_TIMEOUT`. If using `TIMEOUT_PER BLOCK` it is perfectly reasonable to have a `REGION_TIMEOUT` < `LENGTH_TIMEOUT`